src/annotator/util/listener-collection.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @typedef Listener
+ * @prop {EventTarget} eventTarget
+ * @prop {string} eventType
+ * @prop {(event: Event) => void} listener
+ */
+
+/**
+ * Utility that prevents forgetting to remove event listener.
```

Comment:
The wording is perhaps a little strong here. It only avoids failing to remove event listeners if you use it correctly. Perhaps something like "Utility that provides a way to conveniently remove a set of DOM event listeners when they are no longer needed."?

Reply:
Thanks